bot.js

```diff
@@ -219,6 +221,34 @@ client.on("guildMemberUpdate", memberUpdateHandler);
 // Every hour check if the current time is within the ranges
 setInterval(() => {
   const shouldPost = isRedditPostingTime();
+  if (!shouldPost) return;
+    Phin({ url: REDDIT_URL, parse: "json" })
+      .then((res) => {
+          notSticky(res);
```

Comment:
Not a very clear function name, with no context i have no idea what "notSticky" does. Maybe "getFirstNotStickiedPost", or something shorter if you can think of it. Descriptiveness is better than brevity.